cmd/genji/shell/command.go

```diff
@@ -136,3 +140,132 @@ func displaySuggestions(in string) error {
 	fmt.Println()
 	return nil
 }
+
+// dumpTable display the content of the given table as SQL statements.
+func dumpTable(db *genji.DB, t *database.Table) error {
+	// Get table info.
+	ti, err := t.Info()
+	if err != nil {
+		return err
+	}
+
+	fields := ti.FieldConstraints
+	if len(fields) == 0 {
+		fmt.Printf("CREATE TABLE %s;\n", t.Name())
+	} else {
+		fmt.Printf("CREATE TABLE %s(", t.Name())
+		for i, f := range fields {
+			fmt.Printf("%s %s", fields[i].Path, strings.ToUpper(fields[i].Type.String()))
+			if f.IsPrimaryKey {
+				fmt.Printf(" PRIMARY KEY")
+			}
+
+			if f.IsNotNull {
+				fmt.Printf(" NOT NULL")
+			}
+
+			// Don't print the last comma.
+			if i == (len(fields) - 1) {
+				fmt.Println()
+				break
+			}
+
+			fmt.Println(",")
+		}
+
+		fmt.Println(");")
+	}
+
+	// Indexes section.
+	indexes, err := t.Indexes()
+	if err != nil {
+		return err
+	}
+
+	// Display indexes.
+	for _, index := range indexes {
+		u := ""
+		if index.Opts.Unique {
+			u = " UNIQUE"
```

Comment:
```suggestion
			u = "UNIQUE"
```

Reply:
I put a white space too avoids this:
```SQL
...
foo.bar[1].hello BLOBPRIMARY KEY,
foo.a[1][2] TEXTNOT NULL,
...
```

Reply:
This is about the index creation not about the table creation, look below you are running `fmt.Sprintf("CREATE INDEX")`

Reply:
> This is about the index creation not about the table creation, look below you are running fmt.Sprintf("CREATE INDEX")

@tzzed